widget/diagramwidget/README.md

```diff
@@ -0,0 +1,81 @@
+# Fyne DiagramWidget
```

Comment:
Can you also include something into the top level README so it is more discoverable?

Reply:
Moved all of the DiagramWidget documentation into the top-level Readme